README.md

```diff
@@ -1,2 +1,23 @@
 [![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/UxpU_KWG)
-# ServerlessTemplate
\ No newline at end of file
+
+# Software Requirements
+
+Download and install these software if you do not have them locally.
+
+- [AWS CLI](https://aws.amazon.com/cli/)
+
+# Setup
+
+1. Install the Serverless Framework Globally (if not already installed):
+   `npm install -g serverless`
+
+2. Install Dependencies
+   `npm install`
+
+3. Configure AWS Credentials
+   `aws configure`
+
+- AWS Access Key can be created under Identity and Access Management's (IAM) Security Credential to enable sending of programmatic calls to AWS from the AWS CLI
+
+4. Deploy the Services
+   `serverless deploy`
```

Comment:
Perhaps can add this as the last point: user can check the status/info regarding the deployed Lambda function on AWS Lambda console.

Reply:
Done